src/imitation/rewards/reward_nets.py

```diff
@@ -225,10 +225,10 @@ def dtype(self) -> th.dtype:
 
 
 class RewardNetWrapper(RewardNet):
-    """Abstract class representing a wrapper that modifies a `RewardNet`'s functionality.
+    """Abstract class representing a wrapper modifying a ``RewardNet``'s functionality.
```

Comment:
I would say the previous phrasing was "better" in that now it repeats the "-ing" twice which sounds more verbose to me, but I'm not a native English speaker so I might be wrong.

Reply:
I agree, but the previous phrasing was one character over the line limit, which for some reason pre-commit is enforcing more strictly than flake8 was before. :shrug:

Reply:
Makes sense.